tests/frontmatter/unformatted.pug

```diff
@@ -0,0 +1,4 @@
+---
+name: YAML test
```

Comment:
Maybe add a little bit more and unformatted yaml content, so we can also see that it gets formatted

Reply:
Not resolved yet :wink: because you only added pug code, but not further _unformatted_ frontmatter code (e.g. you could add some whitespaces inside the values and add nested keys and lists)